doc/guide/asciidoc/man/cockpit-bridge.adoc

```diff
@@ -0,0 +1,60 @@
+cockpit
+
+cockpit-bridge
+
+1
+
+cockpit-bridge
+
+Cockpit Host Bridge
+
+cockpit-bridge
+
+--help
+
+--packages
+
```

Comment:
This looks really strange -- This should get some more structure, like "NAME", "SYNOPSIS", and proper verbatim formatting and such. But I suppose that's what you meant with "man pages need work".